src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingjavadocmethod/InputMissingJavadocMethodPublicOnly2.java

```diff
@@ -0,0 +1,120 @@
+////////////////////////////////////////////////////////////////////////////////
+// Test case file for checkstyle.
+// Created: 2021
+////////////////////////////////////////////////////////////////////////////////
+package com.puppycrawl.tools.checkstyle.checks.javadoc.missingjavadocmethod;
+
+/* Config:
+ * scope = Scope.NOTHING.getName()
+ */
+public class InputMissingJavadocMethodPublicOnly2 // ignore - need javadoc
+{
+    private interface InnerInterface // ignore - when not relaxed about Javadoc
```

Comment:
Should this be `ignore - when relaxed about javadoc`?  I think if we  were NOT relaxed, we would show violations for these missing javadocs.  Does that make sense?

Reply:
Yes, I thought of resolving this here itself, as a lot of test inputs (for other Javadoc checks also) have these comments, and they are quite confusing. Let me know if I should change these.

Reply:
I think that any comments other than "ok" and "violation" are extraneous in test inputs usually, unless logic for violations in complicated (counting number of occurrences, etc).  Many of the comments in these javadoc checks are because they were used for mutliple tests with different configs; now that you are changing this to one test input per test, these aren't needed.

Reply:
Removed the extra comments.

Reply:
Resolved.